lib/coveralls/api.rb

```diff
@@ -63,9 +63,9 @@ def self.hash_to_file(hash)
 
 		def self.apified_hash hash
 			config = Coveralls::Configuration.configuration
-			if ENV['CI'] || ENV['COVERALLS_DEBUG'] || Coveralls.testing
```

Comment:
Ooops, my bad. That was for testing and I left it accidentally in the commit. Hold on, I'm going to remove it.